### ISSUE TITLE
German Number For Fax on a PC not valid

### ISSUE THREAD

**Issue by cfreak** [googlei18n/libphonenumber#877]
I try to validate this number +4989620603291873 but it fails, but the fax number is valid and exists.

Is that behaviour correct?

**Comment by andyst** [comment]
Thanks for the report.

 - Verified that +4989620603291873 is considered invalid by libphonenumber.appspot.com.
 - That's 14 digits (without the country code), starting with 896206.
 - Current metadata only allows for 89 followed by up to 11 digits. I.e. 13 digits overall, one short.

The latest ITU doc for Germany (from April this year) says that the maximum length for numbers starting with 89 (Munich) is 11, including the 89 prefix.
http://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000510002PDFE.pdf

I.e. we're already more lenient than the ITU doc for the 89 prefix.

Do you have any evidence (government doc or a doc / website from your carrier) to describe / explain the situation?

**Comment by SARG04** [comment]
>Do you have any evidence (government doc or a doc / website from your carrier) to describe / explain the situation?

Federal Network Agency (only in german):
http://www.bundesnetzagentur.de/SharedDocs/Downloads/DE/Sachgebiete/Telekommunikation/Unternehmen_Institutionen/Nummerierung/Rufnummern/ONRufnr/NummernplanOrtsnetzrufnummern.pdf

>Die längerstellige Nutzung von Rufnummern (Anhängen von Ziffern) durch den Teilnehmer 
ist grundsätzlich zulässig. Es wird jedoch auf folgendes hingewiesen:  
a)  Aus der längerstelligen Nutzung durch den Teilnehmer erwachsen keine 
Rechtsansprüche. Dies gilt insbesondere im Zusammenhang mit etwaig erforderlich 
werdenden Rufnummernänderungen und im Zusammenhang mit Portierungen. 
b)  Inwieweit längere Nummern technisch erreichbar sind, richtet sich nach den 
Gegebenheiten bei den an einer Verbindung beteiligten Netzbetreibern. 
c)  Nach der Empfehlung E.164 der Internationalen Fernmeldeunion können Rufnummern in 
Deutschland bis zu 13 Ziffern lang sein (ohne Prefix).  

In short it is basically allowed to add additional numbers to your official number. 
But there is no guarantee that the number can be called.


So likely a company added additional digest to there official number.
I haven't seen any other document about longer numbers.

**Comment by penmetsaa** [comment]
@cfreak,

Thanks for pointing us to the official doc. :)

As per Germany numbering plan (and E.164 format), the phone numbers can not be more than 13 digits (excluding CC). The provided number is of 14 digits and we are not able to connect to the number as well. It can be the case that eventhough you dial additional digits, it connects to same number. This is a know issue, please take a look at our [FAQ page](https://github.com/googlei18n/libphonenumber/blob/master/FAQ.md#but-my-dialled-number-connected-so-isnt-it-valid). As andyst@ mentioned we already support 89 followed by up to 11 digits  i.e. 13 digits overall.

Please feel to reopen the issue if any valid official evidence or phone number that is working is available.
